src/Config_Command.php

```diff
@@ -549,9 +549,6 @@ public function delete( $args, $assoc_args ) {
 						WP_CLI::error( "The {$type} '{$name}' is not defined in the 'wp-config.php' file." );
 					}
 			}
-
-			$config_transformer->remove( $type, $name );
```

Comment:
You've removed the wrong `remove`.

The one that is superfluous is at https://github.com/wp-cli/config-command/pull/52/files#diff-a3ce20ba86d7a7fcd6d399db066ca679R614

Reply:
You're right, I see the error in my own remove. Sorry.

Reply:
No need to be sorry, it was an easy mistake and that's what code reviews are for.

Reply:
@lalaithan Let me know if you have trouble updating this pull request.